docs/team/skills.md

```diff
@@ -0,0 +1,102 @@
+# Skills and ways to contribute
+
+There are many ways that you can contribute to the JupyterHub / Binder projects.
+This page attempts to provide some pathways for community members to learn and
+apply their skills to the team.
+
+## Where to start?
+
+The short answer to this question is: there's no easy answer :-)  there are many
```

Comment:
```suggestion
The short answer to this question is: it depends :-)  there are many
```
what do you think about highlighting already at the start that it depends on what you want to do, your skills, etc aka "it depends". Suggestion is motivated by trying to avoid greeting people with 'it is not easy'. Even though this isn't what the original sentence said, if you read it :)